test/pylib/scylla_cluster.py

```diff
@@ -426,7 +424,10 @@ async def uninstall(self) -> None:
             return
         logging.info("Uninstalling server at %s", self.workdir)
 
-        shutil.rmtree(self.workdir)
+        try:
+            shutil.rmtree(self.workdir)
+        except FileNotFoundError:
```

Comment:
there is ignore_errors why aren't you using it?

Reply:
`ignore_errors` ignores all errors (as far as I understand from the documentation); I just want to ignore `FileNotFoundError`.